packages/react/src/components/cookieConsent/CookieConsent.stories.tsx

```diff
@@ -1,12 +1,8 @@
 import React, { useState } from 'react';
 
-import {
-  Category,
-  CookieData,
-  Provider as CookieContextProvider,
-  SupportedLanguage,
-  useCookieConsentContext,
-} from './CookieConsentContext';
+import { useConsentContext } from './contexts/ConsentContext';
+import { Category, CookieData, SupportedLanguage, useContentContext } from './contexts/ContentContext';
+import { Context } from './contexts/ContextComponent';
```

Comment:
This `Context` is probably going to be also a globally exported function? Therefore I think the name requires more context to it. I would suggest we add the word _Cookie_ in it